### PR TITLE
Updated Formatting for Readme and Plugin Files + Minor Edits

### PR DESCRIPTION
**Pull request by sbdchd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -238,30 +250,38 @@ Change log
...
 * It is now possible to prevent vim-autoformat from overwriting your settings for  `tabstop`, `softtabstop`, `shiftwidth` and `expandtab` in your .vimrc.
 
 ### March 10 2013
+
 * When no formatter is installed or defined, vim will now auto-indent the file instead. This uses the indentfile for that specific filetype.
 
 ### March 9 2013
-The `custom_config` branch has been merged into the master branch.
+
+The j`custom_config` branch has been merged into the master branch.
```
Comment (Chiel92) [inline README.md]: This is a typo :)

**Comment by Chiel92** [comment]
Thanks for this PR. One minor comment, otherwise this looks fine!

**Comment by sbdchd** [comment]
`=~` is kind of ambigious because it relies on the `ignorecase` setting in someone's config.

`=~?` regex matches and always ignores case.

[vint](https://github.com/Kuniwak/vint) can be pretty helpful when checking for weird quarks of vimscript.

**Comment by Chiel92** [comment]
Thanks for this PR!

**Pull request merged**